pkg/controller/backup_deletion_controller.go

```diff
@@ -33,7 +36,8 @@ import (
 	kubeerrs "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/client-go/tools/cache"
 
-	v1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/backup"
```

Comment:
this is already imported just below and aliased as `pkgbackup`

Reply:
I missed that. Thanks!